src/js/input-range/input-range.jsx

```diff
@@ -51,6 +59,7 @@ export default class InputRange extends React.Component {
       disabled: false,
       maxValue: 10,
       minValue: 0,
+      minSize: 1,
```

Comment:
I've changed this to default to null, and use the step size if nothing is passed in. Thoughts?